cardano-launcher/src/Cardano/Shell/Launcher.hs

```diff
@@ -158,16 +158,16 @@ handleDaedalusExitCode
     -> IO DaedalusExitCode
 handleDaedalusExitCode runUpdaterFunction restartWalletFunction = isoTo <<$>> \case
     RunUpdate               -> runUpdate runUpdaterFunction >> runRestart restartWalletFunction
-    -- ^ Run the actual update, THEN restart launcher.
+    -- Run the actual update, THEN restart launcher.
```

Comment:
Any reason for those? Turning Haddock docs into regular comments?

Reply:
The reason why I've deleted those comments is because the hydra build was failing.
https://hydra.iohk.io/build/1153157/nixlog/1
I think this is because it's using haddock syntax within a function is not allowed

```haskell
-- ^ Test
testFoo :: Int -> Int -> Int
testFoo one two = (multiply one) + two
  where
     -- ^ This will make haddock build fail
     multiply = * 3
```